cg/services/order_validation_service/workflows/micriobial_fastq/models/sample.py

```diff
@@ -1,10 +1,10 @@
-from cg.models.orders.sample_base import PriorityEnum
+from cg.models.orders.sample_base import ControlEnum, PriorityEnum
 from cg.services.order_validation_service.constants import ElutionBuffer
 from cg.services.order_validation_service.models.sample import Sample
 
 
 class MicrobialFastqSample(Sample):
```

Comment:
One could consider overriding the parent `Sample`'s `volume` to mark it as a required field as well.